### ISSUE TITLE
[NEO/FORGE] Botania's creative tab is on the first page of the Creative Inventory

### ISSUE THREAD

**Issue by VOLKOUL** [Vazkii/Botania#4576]
### Mod Loader

Forge

### Minecraft Version

1.20.1

### Botania version

Botania-1.20.1-443-FORGE

### Modloader version

NeoForge 1.20.1-47.1.101

### Issue description

Botania's creative tab is on the first page of the Creative Inventory. From what I know this a Forge/NeoForge issue only.

**Comment by anoomolu** [comment]
Duplicate of https://github.com/VazkiiMods/Botania/issues/4562, which was also made by you haha

**Comment by VOLKOUL** [comment]
Yeah, at first I believed it was intentional, but since people pointed out it's a bug I thought making a proper report would be appropriate. (damn, I'm straight up rapping)

**Comment by VOLKOUL** [comment]
Oh, that people was you lol

**Comment by anoomolu** [comment]
https://github.com/VazkiiMods/Botania/issues/4562#issuecomment-1935766929 for some screenshots comparing Forge and Fabric.

**Comment by TheRealWormbo** [comment]
Can't reproduce with 445 on Forge, or 443 on NeoForge for that matter. (Just Botania and its dependencies.)

**Comment by ScoutTheSaneless** [comment]
i can confirm this is an issue i'm having

**Comment by ScoutTheSaneless** [comment]
question, do u also happen to hav Create?

**Comment by VOLKOUL** [comment]
Yeah

**Comment by TheRealWormbo** [comment]
I've seen that happen in ATM9 Skies recently when trying to reproduce a different issue. Is there a smaller reproduction setup, though?

**Comment by anoomolu** [comment]
I'm unable to reproduce this with just Botania on latest Neoforge (47.1.106) fwiw

**Comment by kacho-fuugetsu21** [comment]
I encountered the same problem in forge47.3.22. Another mod in my mod pack also had this problem. After I disabled the adventure module of apotheosis mod, its enchantment module tab was accidentally moved to the first page of the creative mode inventory (I also encountered a similar problem in atm9)
Here is my module list
[modlist.json](https://github.com/user-attachments/files/18398409/modlist.json)

**Comment by TheRealWormbo** [comment]
Update: This still seems to persists into 1.21.1 when combined with an unknown list of other mods.

[edit] Actually, I think I found the issue…
